package.json

```diff
@@ -4,18 +4,40 @@
   "main": "app/app.js",
   "type": "module",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint app/",
+    "lint:fix": "eslint app/ --fix",
+    "format": "prettier --write app/",
+    "prepare": "husky",
+    "lint-staged": "lint-staged"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "description": "",
+  "engines": {
+    "node": ">=20.18.0"
+  },
   "dependencies": {
     "axios": "^1.7.7",
     "compression": "^1.7.4",
     "dotenv": "^16.4.5",
     "express": "^4.21.1",
     "joi": "^17.13.3",
     "nunjucks": "^3.2.4"
+  },
+  "devDependencies": {
```

Comment:
```json
"lint-staged": {
    "*.{js, ts}": [
      "prettier --write",
      "eslint"
    ]
}
```

This can help to run changes only on files you edited.

Reply:
Added